### DISCUSSION TITLE
Rust Analyzer not being detected by helix on OpenSuse Tumbleweed

### DISCUSSION THREAD

**Discussion by 0xEquinox** [helix-editor/helix#9357] [Troubleshooting]
I'm certain that I'm doing something wrong, though I've used helix across windows, fedora, arch, and now tumbleweed and this is the only time I've ever had this issue.

I've installed helix through opi (aur for tumbleweed) and it's on the latest version, working perfectly fine.

I've installed rust-analyzer through rustup by doing `rustup component add rust-analyzer` and it's in the path and working, if I run `rust-analyzer --version` then it prints out: 
![image](https://github.com/helix-editor/helix/assets/69062137/435d4590-c70f-42eb-9ec2-2f2f4a909576)

rust analyzer definitely works and is in the system path, but when I open up helix and edit a rust file it doesn't work. Here you can see both helix health doesn't pick it up and the log file says it's not in path: 
![image](https://github.com/helix-editor/helix/assets/69062137/b62111f6-8067-485d-a511-66c01187d41d)
![image](https://github.com/helix-editor/helix/assets/69062137/56fdb2e0-d380-491c-b120-a3fccf263497)

I also tried building from source and manually adding the binary to my system path, that did not work either. Any help would be much appreciated, I really hate using intelliJ.

**Comment by jw013** [comment]
Assuming you are using the `bash` shell to run `rust-analyzer` and `hx --health`, what happens if you run `type -a rust-analyzer`?
